README.md

```diff
@@ -1,84 +1,126 @@
 ## Table of contents
 
-- [HelloID-Conn-Prov-Target-YsisV2](#helloid-conn-prov-target-ysisv2)
-- [| This connector replaces the current Ysis connector.  |](#-this-connector-replaces-the-current-ysis-connector--)
+- [HelloID-Conn-Prov-Target-Ysis](#helloid-conn-prov-target-Ysis)
   - [Table of contents](#table-of-contents)
   - [Introduction](#introduction)
-  - [Introduction](#introduction-1)
   - [Getting Started](#getting-started)
     - [Prerequisites](#prerequisites)
     - [Connection settings](#connection-settings)
+    - [Correlation configuration](#correlation-configuration)
+    - [Field mapping](#field-mapping)  
     - [Remarks](#remarks)
+      - [Concurrent actions to 1](#Concurrent-actions-to-1)
       - [`PUT` method for all update actions](#put-method-for-all-update-actions)
       - [Full update within the _update_ lifecycle action](#full-update-within-the-update-lifecycle-action)
-      - [Discipline and the Ysis-initals are stored in `$aRef`](#discipline-and-the-ysis-initals-are-stored-in-aref)
       - [Archiving an Ysis-account](#archiving-an-ysis-account)
-    - [Mapping](#mapping)
-    - [Correlation](#correlation)
-    - [Conditional Event](#conditional-event)
+      - [Conditional event for notification when discipline changes](#conditional-event)
+      - [Fields "Beroep" and "Opmerking" are cleared](#fields-beroep-and-opmerking-are-cleared)
+      - [End date must be cleared](#end-date-must-be-cleared)
+      - [Username must be unique in Ysis](#username-must-be-unique-in-ysis)
   - [Getting help](#getting-help)
   - [HelloID Docs](#helloid-docs)
 
 ## Introduction
 
-The HelloID-Conn-Prov-Target-YsisV2 connector creates and updates user accounts within Ysis. The Ysis API is a SCIM based (http://www.simplecloud.info) API and has some limitations for our provisioning process. For more information you can check the Ysis SCIM documentation (https://apihelp.gerimedica.nl/category/scim/).
+The HelloID-Conn-Prov-Target-Ysis is a _target_ connector that creates and updates user accounts, modules and roles within Ysis.
+
+Ysis provides a set of SCIM (http://www.simplecloud.info) based API's. The HelloID connector uses the API endpoints listed in the table below.
+
+| Endpoint | Description |
+| -------- | ----------- |
+| /cas/oauth/token | Generate an authorization token 
+| /gm/api/um/scim/v2/users | Search, create or update an account; assign or remove modules or roles to account |
+| /gm/api/um/scim/v2/roles | Get role data; default roles and custom roles |
+ 
+The API has a limitation requiring the complete account object to be sent when updating an account. For further details, refer to the Ysis SCIM documentation: Ysis SCIM Documentation. (https://apihelp.gerimedica.nl/category/scim/
+
+> [!IMPORTANT] Changing the discipline of an existing account is not supported. If a discipline change is attempted during the update life-cycle, a conditional event is triggered, sending an email notification to the Ysis administrator.
```

Comment:
Use enter after [!IMPORTANT] the readme on github is now showing '[!IMPORTANT]' instead of the Important markdown